scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -0,0 +1,84 @@
+version 1.0
+
+import "GvsPrepareCallset.wdl" as GvsPrepareCallset
+import "GvsExtractCallset.wdl" as GvsExtractCallset
+
+# Workflow used by AoU to extract variants for a given cohort of sample_names
+
+workflow GvsExtractCohortFromSampleNames {
+
+  input {
+    File cohort_sample_names
+    String query_project
+    String gvs_project
+    String gvs_dataset
+    String fq_gvs_extraction_cohorts_dataset
+    String fq_gvs_extraction_destination_dataset
+    String fq_gvs_extraction_temp_tables_dataset
```

Comment:
You can make these 2 optional since they are defaulted in the called workflow.
also I don't see `fq_gvs_extraction_cohorts_dataset` used anywhere. should it be removed?

Reply:
good catch. removed.

regarding the two optionals though, do you mean the arguments on lines 16 and 17? those are passed directly into Prepare

Reply:
yes. if you make `fq_gvs_extraction_destination_dataset` and `fq_gvs_extraction_temp_tables_dataset` optional (and still pass them into the prepare) then the caller does not have to supply the input. If an optional that has not been set is passed to a workflow/task they will be treated as None (and will therefore get the default values). The only reason not to make them optional is if you know the defaults won't work for you.